data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -325,6 +325,74 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/message]
     [/event]
 
+    # Triggers the first time that the player has two healers
+    [event]
+        name=recruit
+        [filter]
+            side=1
+            type=Elvish Shaman
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                ability_type=heals
+                count=2
+            [/have_unit]
+        [/filter_condition]
+
+        [store_unit]
+            [filter]
+                ability_type=heals
+                [not]
+                    id=$unit.id
+                [/not]
+                {EVERYWHERE}
+            [/filter]
+            variable=other_healer
+        [/store_unit]
+
+        [if]
+            [variable]
+                name=other_healer.type
+                equals=Elvish Shaman
+            [/variable]
+            [then]
+                [message]
+                    speaker=$unit.id
+                    # po: both speaker and $other_healer are elvish shamans.
+                    # If there are allied sides then this isn't accurate, but at this stage of the tutorial allied sides haven't been introduced yet.
+                    message= _ "Both Shaman $other_healer.name and I can heal other units. However, each unit can only receive healing from one healer — even if we’re both next to a unit then that unit will still only heal 4 hitpoints per turn."
```

Comment:
This uses a variable name in a translated string, which https://forums.wesnoth.org/viewtopic.php?f=8&t=11445 says not to. Is that a problem? It might be fine since it's in the nominative case?

Maybe change `heal` to `be healed`. I realize it's correct as it is, but making the change will prevent players from incorrectly reading `heal` as a transitive verb there, as though a unit that's adjacent to a shaman can heal _other_ units.

Reply:
Good point. I wonder what about `# po: There's no need to use $other_healer.name in the translated string, leave it out if that makes the translation better`? I've posted that question to the forums.

Will make the change to `be healed`.